sdk/state/open_test.go

```diff
@@ -0,0 +1,47 @@
+package state
+
+import (
+	"testing"
+
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/network"
+	"github.com/stretchr/testify/require"
+)
+
+func TestProposePayment_valid_asset(t *testing.T) {
```

Comment:
adding 1 unit test here, but more exhaustive unit tests will be added as part of [this ticket](https://github.com/stellar/experimental-payment-channels/issues/70)